### DISCUSSION TITLE
container healthcheck support

### DISCUSSION THREAD

**Discussion by advnpzn** [containerd/nerdctl#3149] [General]
Hi, I just wanted to know whether there is already a plan for adding healthcheck support to nerdctl.
I was trying to use `healthcheck` in a compose file and deploy it using `nerdctl compose`, but I got `WARN[0000]` for health check. I had gone through the CLI reference sheet and it seems like the implementation is not done yet for this.

**Comment by JWDobken** [comment]
Now it simply ignores these:

```cmd
❯ nerdctl compose -f kong-compose.yml up --remove-orphans
WARN[0000] Ignoring: service kong-database: [HealthCheck] 
WARN[0000] Ignoring: service kong: depends_on: kong-database: [Required] 
WARN[0000] Ignoring: service kong: depends_on: kong-database: condition service_healthy 
```

Also check this issue: https://github.com/containerd/nerdctl/issues/2386
